### DISCUSSION TITLE
Support constant fields in query stanzas in dwca export definitions

### DISCUSSION THREAD

**Discussion by benanhalt** [specify/specify7#5560] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
This would facilitate the export of measurement and fact data and other key-value type data. The key would be the constant field and the value would come from a regular query field with a not-empty filter. There would be one query stanza for each possible key.

**Reported By**
Andy Bentley in re to KU Birds

**Comment by maxpatiiuk** [comment]
Current workarounds:
- Add a relationship to the collection/discipline/division/institution table, and use one of the fields in that table to store the static value
- Create a formatter that has a static value as a separator (would be made simpler after https://github.com/specify/specify7/issues/1959 and https://github.com/specify/specify7/pull/2796)

**Comment by grantfitzsimmons** [comment]
This is an issue I encounter regularly. Silvia at CSIC and Katherine at UWFC wanted to be able to add static values as well.
